src/Templates/EnumTemplate.cshtml

```diff
@@ -10,24 +11,43 @@
 @WrapComment("// ", Model.Documentation)
 type @Model.Name string
 
+
 @EmptyLine
-@if (values.Any())
+@if (Model.Values.Any())
```

Comment:
Is the above declaration of values no longer needed?

Reply:
yes ill remove